dotnet/test/Svg.test.cs

```diff
@@ -0,0 +1,40 @@
+namespace StackExchange.StacksIcons.Test;
+
+public class SvgTest
+{
+    [Fact]
+    public void FindsEmbeddedResources()
+    {
+        var icon = Svg.Accessibility;
+        var spot = Svg.Spot.Ads;
+        Assert.False(string.IsNullOrWhiteSpace(icon.Value));
+        Assert.False(string.IsNullOrWhiteSpace(spot.Value));
+    }
+
+    [Theory]
+    [InlineData(null, null)]
+    [InlineData("", "")]
+    [InlineData("foo-class", null)]
+    [InlineData(null, "foo-title")]
+    [InlineData("foo-class", "bar-title")]
+    public void With(string? cssClass, string? title)
```

Comment:
Thanks for adding the test! 🙂
I think we are good for now, if we refactor the exception handling behavior it might be worth having a unit test for that too.